include/fmt/chrono.h

```diff
@@ -582,8 +582,9 @@ template <typename Period> FMT_CONSTEXPR inline const char* get_units() {
   if (std::is_same<Period, std::tera>::value) return "Ts";
   if (std::is_same<Period, std::peta>::value) return "Ps";
   if (std::is_same<Period, std::exa>::value) return "Es";
-  if (std::is_same<Period, std::ratio<60>>::value) return "m";
+  if (std::is_same<Period, std::ratio<60>>::value) return "min";
   if (std::is_same<Period, std::ratio<3600>>::value) return "h";
+  if (std::is_same<Period, std::ratio<86400>>::value) return "d";
```

Comment:
Please add a test case to https://github.com/fmtlib/fmt/blob/f76603f21ea75bcd8d0007d7599cac75862b13b9/test/chrono-test.cc#L425C19-L425C33.

Reply:
Well, according to the test, I have no clue how to use `__cpp_lib_chrono` …

Reply:
```c++
#if defined(__cpp_lib_chrono) && __cpp_lib_chrono >= 201907L
```